pkg/cluster/cluster.go

```diff
@@ -94,7 +94,11 @@ func new(kclient *unversioned.Client, name, ns string, spec *spec.ClusterSpec, s
 		status:    &Status{},
 	}
 	if isNewCluster {
-		var err error
+		err := c.createClientServiceLB()
+		if err != nil {
+			// todo: do not panic!
```

Comment:
I do not quite understand. Do you want me to simply add a TODO into the message? Or you want me to address the TODO in this PR? I leave a todo here since we panic all over the place already. It requires a dedicated effort to address the panic problem.